wordpress-mobile/WordPress-iOS
# Removes an invalid relationship from the Comment entity to ReaderSiteTopic.sites

### To test:
1. Pull PR
2. Perform a clean build
3. Click on the warning: ```The inverse relationship for Comment.sites does not reciprocate an inverse relationship [2]``` 
4. When looking at the details it should not mention model version `WordPress 110`
5. Run the app
6. Verify you are not logged out / the model is correctly migrated
7. Tests are 🟢 

Sadly the warnings for the previous models will continue to exist. 

PR submission checklist:
